integration/common.sh

```diff
@@ -29,6 +29,20 @@ waitForServerStartup() {
   sleep 2
 }
 
+pickUnusedPort() {
+  local base=$1
```

Comment:
I was trying to avoid re-iterating over known-used ports, but I guess it's fast & safe enough so: Done.